papers/moore-pyqtgraph/metadata.yaml

```
@@ -0,0 +1,37 @@
+---
+abstract:
+  - PyQtGraph is a plotting library with high performance, cross-platform support and interactivity as its primary objectives. These goals are achieved by connecting the Qt GUI framework and the scientific Python ecosystem. The end result is a plotting library that supports the use of python and NumPy arrays to drive interactive visualizations on all major operating systems.
```

I was unclear if you meant you have supported for NumPy arrays as well as the builtin `array` module in Python. I assumed you just mean NumPy, but if not it might be helpful to clarify.

```suggestion
  - PyQtGraph is a plotting library with high performance, cross-platform support and interactivity as its primary objectives. These goals are achieved by connecting the Qt GUI framework and the scientific Python ecosystem. The end result is a plotting library that supports the use of NumPy arrays to drive interactive visualizations on all major operating systems.
```

---

Thanks for making this clearer to me!